src/main/java/Task.java

```diff
@@ -0,0 +1,38 @@
+public class Task {
+    private String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public boolean getStatus() {
+        return isDone;
+    }
+
+    public void setAsDone() {
+        this.isDone = true;
+    }
+
+    public void setAsNotDone() {
```

Comment:
Instead of setAsNotDone and setAsDone you can just use one method setDoneStatus and take an input to change the value of isDone